docs/usage/visualize.ipynb

```diff
@@ -563,7 +616,13 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
```

Comment:
I'm gonna add a separate PR that will automatically remove this metadata, just like https://github.com/ComPWA/compwa.github.io/pull/279.